spaceR/spaceR/Controllers/SignupViewController.swift

```diff
@@ -30,7 +30,10 @@ class SignupViewController: UIViewController {
            let safeEmail = self.userEmail.text, !safeEmail.isEmpty,
            let safePassword = self.userPassword.text,
            !safePassword.isEmpty {
-            setSignupModelAttributes(firstName: safeFirstName, lastName: safeLastName, email: safeEmail, password: safePassword)
+            setSignupModelAttributes(firstName: safeFirstName,
+                                     lastName: safeLastName,
+                                     email: safeEmail,
+                                     password: safePassword)
             return true
```

Comment:
So this logic needs to be moved to the `ViewModel` and then handled with your `delegate`

Reply:
cool